src/credentials/DeviceAttestationConstructor.cpp

```diff
@@ -33,6 +33,12 @@ enum : uint32_t
     kAttestationNonceTagId         = 2,
     kTimestampTagId                = 3,
     kFirmwareInfoTagId             = 4,
+
+    kCsr             = 1,
+    kCsrNonce        = 2,
+    kVendorReserved1 = 3,
+    kVendorReserved2 = 4,
+    kVendorReserved3 = 5,
```

Comment:
Suggest not mixing here, and create a new enum. Overall, it would probably be better to actually have named enums rather than anonymous enum within the `Credentials` namespace

Reply:
Addressed in f522e7e